mod/tagrm.php

```diff
@@ -57,6 +74,11 @@ function tagrm_content(App $a)
 		// NOTREACHED
 	}
 
+	if ($a->argc == 3) {
+		update_tags($a->argv[1], [hex2bin(notags(trim($a->argv[2])))]);
```

Comment:
When is this form used? I thought hex-converted tags were separated by commas? How does it go through `hex2bin()`?

Reply:
Ohh your right the order of call is wrong! Is has to be `notags(trim(hex2bin($a->argv[2])))`.
It is called be you press the `(X)` behind the tag.